Testlio-Automation/testlio-automation-templates
# GREEN-964 & GREEN-967: Implemented library for building fake test packages. Implemented basic and stress tests.

#### JIRA
[GREEN-964](https://testlions.atlassian.net/browse/GREEN-964)
[GREEN-967](https://testlions.atlassian.net/browse/GREEN-967)

#### Related PRs
- https://github.com/Testlio-Automation/java-automation-framework/pull/5

#### Background and Solution
- Implemented library for building fake test packages
- Implemented basic and stress tests

UPD: common classes allowing to build tests moved to https://github.com/Testlio-Automation/java-automation-framework, same tests implemented for `testlio-mobile-testng-template`

#### Testing Strategy
- Check that packaging with `basic`, `stress` and `login` TestNG suites works (use command `./package.sh <suite name>`)
- Verify on platform, that packages executing properly

Compiled test packages (desktop): https://drive.google.com/drive/folders/1IQMnWmNOoSo2kwDFTYnK7g-wbr17c7R1
Compiled test packages (mobile): https://drive.google.com/drive/folders/1xNI7601rtHeWUEhtjm3kRR2zbxyOCMT7

Example runs on prod (desktop):
- `basic`: https://app.testlio.com/tmt/project/testlio-sandbox-playground/runs/2299
- `stress`: https://app.testlio.com/tmt/project/testlio-sandbox-playground/runs/2297

Example runs on prod (mobile):
- `basic`: https://app.testlio.com/tmt/project/testlio-sandbox-playground/runs/2307

#### Templates Affected
 - [ ] Mobile Java TestNG
 - [ ] Mobile Node 
 - [x] Web Java TestNG
